CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/compose/ComposeViewModel.kt

```diff
@@ -3,10 +3,10 @@ package co.nimblehq.coroutine.ui.screens.compose
 import androidx.compose.runtime.State
 import androidx.compose.runtime.mutableStateOf
 import androidx.lifecycle.viewModelScope
```

Comment:
Please remove `androidx.lifecycle.viewModelScope` also for `kotlinx.coroutines.launch` 🙏